config/routes.rb

```diff
@@ -635,10 +635,12 @@
 
       resources :notes, only: %i[index show new create], as: :application_choice_notes
 
-      resources :interviews, only: %i[new index], as: :application_choice_interviews do
+      resources :interviews, only: %i[new edit index], as: :application_choice_interviews do
         collection do
           get '/new/check', to: 'interviews#check'
           get '/confirm', to: 'interviews#commit'
+          get '/check/:id', to: 'interviews#check', as: :check
```

Comment:
These should be in the member block as they apply to a single interview, and the update should be a POST

```suggestion
          get :check
          post :update
```

Reply:
:+1: Should `/confirm` also be moved to as a POST? I didn't think to question why this was a GET at the time.

Reply:
Yes, it should be a post!

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3993/commits/00bd67a555c31f91758853fd2daf472b767f1923

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3993/commits/00bd67a555c31f91758853fd2daf472b767f1923